Remove bundler gem helpers to avoid conflict with git release task (#65)
cipherstash/activestash: README.md

Then, run `rake spec` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.

To install this gem onto your local machine:

> 1. Build the gem by running `gem build activestash.gemspec`.

Reviewer: ```suggestion
1. Build the gem by running `rake gem`.
```

Ah, this might be why your gems were ending up in the repo root, rather than `/pkg`.

Author: yep you're right, using rake gem, the file goes into /pkg. I'll update the readme and also remove .gem from the gitignore file.

> 2. This will create a gem file (`active_stash-x.x.x.gem`) in the root of this directory

Reviewer: ```suggestion
2. This will create a gem file (`active_stash-x.x.x.gem`) in the root of this repository
```

Author: I updated this to refer to the./pkg folder as the install location instead of the root of the repo.

> 2. Install the gem by running `gem install` with the gem file name, e.g `gem install active_stash-x.x.x.gem`

Reviewer: ```suggestion
3. Install the gem by running `gem install` with the gem file name, e.g `gem install active_stash-x.x.x.gem`
```

I don't mind the "use all `1.`" markdown style, but it's best to be consistent across a given list.
